### PR TITLE
[master] Remove online tarball build from CI in favor of enforced offline build.

### PR DESCRIPTION
**Pull request by crummel**
Skip CI please.

### CODE REVIEW

**Comment by crummel** [comment]
@dotnet-bot test CI please.

**Comment by crummel** [comment]
@dotnet-bot test CI please.

File: netci.groovy [hunk 1]
```diff
@@ -109,53 +109,6 @@ def addPushJob(String project, String branch, String os, String configuration)
...
-        Utilities.addGithubPushTrigger(newJob);
-      }
-
-    }
-  }
-}
-
 // Tarball builds that are enforced offline with unshare
 [true, false].each { isPR ->
   ["RHEL7.2"].each { os->
```
Comment (dagood) [inline netci.groovy]: Could you add CentOS here? I forgot that we only had RHEL7.2 here, and I think we should try to keep making and building the tarball on both.

Comment (crummel) [reply netci.groovy]: Sure, I will try it out with the CentOS docker image.

**Comment by crummel** [comment]
@dotnet-bot test CI please

**Pull request merged**